python/interpret_community/explanation/explanation.py

```diff
@@ -1178,7 +1204,7 @@ def _create_local_explanation(expected_values=None, classification=True, explana
         kwargs[ExplanationParams.EXPECTED_VALUES] = expected_values
     if classification:
         mixins.append(ClassesMixin)
-    if classification:
+        kwargs['num_classes'] = len(kwargs['local_importance_values'])
```

Comment:
use ExplainParams constant

Reply:
I have that already marked as remaining work in the PR description

Reply:
oh, didn't notice that, should have read the PR description first!